### PR TITLE
Solved

### PR DESCRIPTION
**Pull request by AlexGrytsai**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_detail(request: HttpRequest, pk: int) -> Response:
+    movie = get_object_or_404(Movie, pk=pk)
+    if request.method == "GET":
+        serializer = MovieSerializer(movie)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    elif request.method == "PUT":
+        serializer = MovieSerializer(movie, data=request.data)
+        if serializer.is_valid():
```
Comment (vasylhnatiuk) [inline cinema/views.py]: use if serializer.is_valid(raise_exception=True)

File: cinema/views.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+@api_view(["GET", "POST"])
+def movie_list(request: HttpRequest) -> Response:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(request.data)
+        if serializer.is_valid():
```
Comment (vasylhnatiuk) [inline cinema/views.py]: use if serializer.is_valid(raise_exception=True)

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
delete .idea from pull ruquest

**Review by YukihiroSM — APPROVED** [review]
Looks good to me! Nice job!

**Pull request closed without merging**